### DISCUSSION TITLE
react-native-screens:compileDebugKotlin FAILED

### DISCUSSION THREAD

**Discussion by Bragadheeshvar** [software-mansion/react-native-screens#1187] [Q&A] [answered]
Getting the attached error when I run the native app:

> Task :react-native-screens:compileDebugKotlin FAILED
w: Java source root points to a non-existent location: C:\Users\bragadeeshvar.gk\OneDrive - ASG Technologies Group
w: Java source root points to a non-existent location:  Inc\Desktop\WorkSpace\ReactNative\React-Native-Navigation6\node_modules\react-native-screens\android\build\generated\source\buildConfig\debug
w: Runtime JAR files in the classpath should have the same version. These files were found in the classpath:
    C:/Users/bragadeeshvar.gk/.gradle/caches/transforms-3/6786ed219b01aac6d7fac2224360f0ad/transformed/jetified-kotlin-stdlib-jdk8-1.4.10.jar (version 1.4)
    C:/Users/bragadeeshvar.gk/.gradle/caches/transforms-3/0b89a6e271d5397d75101de110d10151/transformed/jetified-kotlin-stdlib-jdk7-1.5.20.jar (version 1.5)
    C:/Users/bragadeeshvar.gk/.gradle/caches/transforms-3/6f601df8028ce88b89197e8db9061d3b/transformed/jetified-kotlin-stdlib-1.5.20.jar (version 1.5)
    C:/Users/bragadeeshvar.gk/.gradle/caches/transforms-3/a47876ccfbe98b22d9c60c8cfded68df/transformed/jetified-kotlin-stdlib-common-1.5.20.jar (version 1.5)
w: Some runtime JAR files in the classpath have an incompatible version. Consider removing them from the classpath
93 actionable tasks: 93 executed
Note: C:\Users\bragadeeshvar.gk\OneDrive - ASG Technologies Group, Inc\Desktop\WorkSpace\ReactNative\React-Native-Navigation6\node_modules\@react-native-async-storage\async-storage\android\src\main\java\com\reactnativecommunity\asyncstorage\AsyncStorageModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: C:\Users\bragadeeshvar.gk\OneDrive - ASG Technologies Group, Inc\Desktop\WorkSpace\ReactNative\React-Native-Navigation6\node_modules\@react-native-async-storage\async-storage\android\src\main\java\com\reactnativecommunity\asyncstorage\AsyncStoragePackage.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
e: C:\Users\bragadeeshvar.gk\OneDrive - ASG Technologies Group, Inc\Desktop\WorkSpace\ReactNative\React-Native-Navigation6\node_modules\react-native-screens\android\src\main\java\com\swmansion\rnscreens\ScreenStackHeaderConfig.kt: (376, 23): Unresolved reference: BuildConfig

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-screens:compileDebugKotlin'.
> Compilation error. See log for more details

**Comment by kacperkapusciak** [comment, marked as answer]
This issue was fixed in v3.8.0 in https://github.com/software-mansion/react-native-screens/pull/1130

You can see the connected issue: https://github.com/software-mansion/react-native-screens/issues/1109
